### DISCUSSION TITLE
index.md の見出しの追加（記事の作成も）

### DISCUSSION THREAD

**Discussion by graycat27** [freeserverproject/freeserver-wiki#115] [Ideas]
`生活情報` みたいな括りで、以下のような情報の記事がほしい
- mobトラップ（一覧記事。もし余力あれば個別記事も？）
- 公共の農場
- 公共の牧場
- 広域移動手段（鉄道・ネザー/地上の主要道）

**Comment by Nakatai-0322** [comment]
了解です。
Mobトラップの記事・広域移動手段を同時並行で進めやす。

**Comment by tsutoringo** [comment]
これをするのであれば #10 を先に終わらせて
作ったうえで座標とかはそっちに飛ばして案内するのがベスト

**Comment by Nakatai-0322** [comment]
了解なり。

**Comment by Nakatai-0322** [comment]
Mobトラップに関して，現在私ローカルで作業しています。（一応報告）

**Comment by Nakatai-0322** [comment]
{報告}
Mobトラップの記事を作成しました。情報量が少なすぎるので追加・情報提供お願いします。
#44

**Comment by Nakatai-0322** [comment]
> {報告}
> Mobトラップの記事を作成しました。情報量が少なすぎるので追加・情報提供お願いします。
> #44 

追記
#45 world/に移動させました。

**Comment by Nakatai-0322** [comment]
- [x] Mobトラップ
- [ ] 公共農場
- [ ] 公共牧場
- [ ] 広域移動手段

**Comment by A-B-S-I** [comment]
施設情報のディレクトリを新規作成して、記事を増やしていく方針で
